README.md

```diff
@@ -17,44 +17,34 @@ Note: The PRM and Bimanual reproductions do not yet work on Deepnote and the UAV
 
 ## Running locally
 
-### Installing Dependencies
-This code depends on [Drake](https://drake.mit.edu), specifically its Python bindings. To install Drake bindings follow the instruction on [Drake's Installation Page](https://drake.mit.edu/installation.html). Any of the installation methods listed should work.  You can check that the installation was sucessful by following the instruction [here](https://drake.mit.edu/python_bindings.html#using-the-python-bindings).
+### Setup Docker
+We provide a [dockerfile](docker/deepnote/Dockerfile) with a custom build of [Drake](https://drake.mit.edu) that entails sampling based planners for the comparison.
 
-We have used Mosek to solve most of the examples. To solve using Mosek, you'll need to give Drake access to a Mosek license file as described [here](https://drake.mit.edu/bazel.html#mosek). Mosek provides a personal academic license for free.
-
-You will also need to install `gcs` and its dependencies. You can do this by running
-```
-pip install -r requirements.txt
+Build the docker:
```

Comment:
I think that rather than tell them how to build docker, we should host it and tell them how to fetch it?  It looks like you have it already on dockerhub wrangelvid/drake:gcs-science-robotics ?

Reply:
Good point! I put the dockers together so only pulling is required.